functions/interface.php

```diff
@@ -107,7 +106,7 @@ function admin_notices() {
 
 				<?php foreach ( $notices['server_config'] as $notice ) : ?>
 
-					<li><?php echo wp_kses_data( $notice ); ?></li>
+					<li><?php echo print_whitelist_html( $notice ); ?></li>
```

Comment:
Yeah a too

Sent from my iPhone

> On 17 Jun 2016, at 10:59, Dasha Luna <notifications@github.com> wrote:
> 
> In functions/interface.php:
> 
> > @@ -107,7 +106,7 @@ function admin_notices() {
> >  
> >  				<?php foreach ( $notices['server_config'] as $notice ) : ?>
> >  
> > -					<li><?php echo wp_kses_data( $notice ); ?></li>
> > +					<li><?php echo print_whitelist_html( $notice ); ?></li>
> and a? There are lots of cases where we refer either to BWP docs or PHP docs. However, in some cases we want to allow URL to be localised and in some now. How do we tackle that?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>